pitch.md

```diff
@@ -1,60 +1,224 @@
-# Project Title (replace me)
+# > Prompt
 
 ## Describe your project in five words
 
+*Greater Digital Cueing for Theater*
+
 ## Audience
 _Who are you helping, with your language? What is that person's
-experience like now? How would their experience be better if you could help 
+experience like now? How would their experience be better if you could help
 them?_
 
+For live theater productions, stage managers produce and use a *prompt book*
+that includes actor blocking<sup>1</sup>, lighting, sound, and scenic cues<sup>2</sup>.
+
+Wikipedia nicely describes the essence of a prompt book:
+
+> The prompt book, also called transcript, the bible or sometimes simply
+> "the book," is the copy of a production script that contains the information
+> necessary to create a theatrical production from the ground up. It is a
+> compilation of all blocking, business, light, speech and sound cues, lists of
+> properties, drawings of the set, contact information for the cast and crew,
+> and any other relevant information that might be necessary to help the
+> production run smoothly and nicely.
+
+While there are many important components of the prompt book, this project
+aims to help stage manager's digitze their cue notes making them searchable,
+summarizable, and editable for not only the stage manager, but the rest of the
+production team&mdash;including the designers who may alter cues many times
+before deciding on a final position for the cue. In the below picture, on
+the right side, you can see how these cues are traditionally recorded: in pencil
+on paper in the margin of the script:
+
+![Prompt Book Example](promptBookExample.jpg)
+
+During a production, *each* of the designers will create cue sheets that
+look something like:
+
+| Cue # | Page # | Cue Name/Description | Location                 |
```

Comment:
Perhaps--however, I don't think it's easy to type with the appropriate formatting 😞 so it will likely be something that contains all this elements, but a more friendly typed structure.